Dockerfile

```diff
@@ -1,6 +1,16 @@
 FROM rethinkdb:2.3
 
-RUN apt-get update
+# From http://stackoverflow.com/a/38553499
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
```

Comment:
Ah, that makes sense. Thanks for explaining!

@sbellem is our local Docker expert and he's away on holidays for a little while longer, so don't worry if he doesn't review this PR right away.